crates/swc_ecma_minifier/src/compress/optimize/inline.rs

```diff
@@ -517,6 +518,18 @@ where
                                     return;
                                 }
 
+                                for i in collect_infects_from(
+                                    &f.function,
+                                    AliasConfig {
+                                        marks: Some(self.marks),
+                                        ignore_nested: false,
+                                    },
+                                ) {
+                                    if let Some(usage) = self.data.vars.get_mut(&i) {
+                                        usage.ref_count += 1;
+                                    }
+                                }
+
```

Comment:
As we only inline very simple functions, this shouldn't be a big performance penalty